README.md

```diff
@@ -212,4 +212,8 @@ Starting with version 2.8, there is the `ide` flavor of the images, which comes
 2.8-php7.3-ide
 ```
 
+`IDE_PASSWORD`
+
+Store you preffered password into this variable, if you need password protected IDE environment
```

Comment:
Store your preferred password in this variable if you need to password protect the IDE environment.